static/src/javascripts/projects/common/modules/discussion/api.ts

```diff
@@ -0,0 +1,71 @@
+import config from 'lib/config';
+
+type Id = number | string;
```

Comment:
Can the same id be a number or string for legacy reasons?
Or are we using this type to cover two different id types in which case would it be better to have (for example) `commentId: string` and `discussionId: number` ?

Reply:
That’s a good question. I got the `string | number` union type from [the Flow archive](https://github.com/guardian/frontend/blob/6165f83791f3360fa7dc25271fc68d3da1d97f87/static/src/javascripts.flow.archive/projects/common/modules/discussion/api.js#L41). Not sure if we can confirm it should be one or the other…